src/identify.js

```diff
@@ -66,23 +66,26 @@ exports.exec = (rawConn, muxer, pInfo, callback) => {
           callback(null, otherPInfo)
         }))
 
-      const obsMultiaddr = rawConn.getObservedAddrs()[0]
+      rawConn.getObservedAddrs((err, addrs) => {
+        if (err) {}
```

Comment:
This will change with the extraction of `Identify` (happening today or tomorrow in a separate PR), but yes, we can return for now.